requirements-base.txt

```diff
@@ -3,6 +3,7 @@ backports.functools-lru-cache==1.6.1
 backports.ssl-match-hostname==3.7.0.1
 beautifulsoup4==4.9.1
 cffi==1.14.3
+click==8.0.1
```

Comment:
Core has a requirement of click 7.1.2, which is causing the conflict that breaks the tests. We don't use click directly, so it must have been introduced as a dependency. Try upgrading it in core and if that doesn't work, try removing this requirement and see if we can use 7.1.2 here.